docs/build/build-hrmp-channels.md

```diff
@@ -62,11 +62,11 @@ review and ultimately vote:
 1.  A request on what the proposal aims to do (opening an HRMP channel with Statemint);
 2.  The use cases this channel will support for your chain;
 3.  Technical details of the proposal, including proposal parameters and technical details of this
-    call (On Kusama, most proposals were designed as a batchAll calls) :
+    call (On Kusama, most proposals were designed as a batchAll calls):
 
-    - A force transfer from Polkadot treasury to Statemint as deposit to accept and open an HRMP
-      channel with your chain;
-    - Send XCM message to Statemint to execute a transaction with superuser (root) permission.
+    - A `forceTansfer` of 20 DOT from the Treasury to Statemint sovereign account.
+    - A `forceOpenhrmpchannel` from Statemint (1,000) to ParaID.
```

Comment:
I am a little confused - is this a general set of rules for HRMP openings? Or is it specific to the linked proposal only?